packages/wrangler/src/api/integrations/platform/index.ts

```diff
@@ -103,27 +111,37 @@ export async function getPlatformProxy<
 >(
 	options: GetPlatformProxyOptions = {}
 ): Promise<PlatformProxy<Env, CfProperties>> {
+	const experimentalRemoteBindings = !!options.experimental?.remoteBindings;
+
 	const env = options.environment;
 
 	const rawConfig = readConfig({
 		config: options.configPath,
 		env,
 	});
 
-	const miniflareOptions = await run(
-		{
-			MULTIWORKER: false,
-			RESOURCES_PROVISION: false,
-			// TODO: when possible remote bindings should be made available for getPlatformProxy
-			REMOTE_BINDINGS: false,
-		},
-		() => getMiniflareOptionsFromConfig(rawConfig, env, options)
+	let remoteProxySession: RemoteProxySession | undefined = undefined;
+	if (experimentalRemoteBindings && rawConfig.configPath) {
+		const a = await maybeStartOrUpdateRemoteProxySession(
```

Comment:
Does this need to use `maybeStartOrUpdate()` given it always passes null as the previous session?

Reply:
No it does not, but I find it to be much more convenient than having to manually parse the config, get the bindings, convert them and pass them onto `startRemoteProxySession` 🤔 

I can do it that way if you prefer but `maybeStartOrUpdateRemoteProxySession` with a `null` second argument basically does that exactly the above (which actually makes me wonder if maybe the function should be renamed and suggested more broadly? I think that most of the time this might just be what users might need? 🤔)

Reply:
PS: I didn't mean to use `a` as a variable name, I need to change that 🤦

Reply:
Ah, fair enough. In that, sounds good (although maybe make the second argument optional?)

Reply:
Second argument made optional 🙂👍